EIPS/eip-4762.md

```diff
@@ -268,6 +254,13 @@ class AccessSubtree(Container):
     elements: BitVector[256]
 ```
 
+## Block-level operations
+
+* Precompile accounts, system contract accounts and slots of a system contract that are accessed during a system call are considered warm.
```

Comment:
just to reconfirm this is only for the system call?

also system call is:
 i) system updates not related to a tx
 ii) a precompile executing system call code  (blockhash)

Reply:
system calls, right now, are 2935 (post osaka), 4788 and 7002. whatever stuff they use that get accessed, should be considered warm.

I am actually close to reconsidering that, it's a special case.

Reply:
Post-Osaka-2935 has a long history. I'm not sure accesses to that history via the system contract should be considered warm?

Reply:
From what I understand the intention is here, "system call" refers to any instruction (e.g: BLOCKHASH) or system contract execution called during a tx.

So "system call" has to contemplate both instructions and contracts, thus this is the reason why this concept emerges? (just trying to double-check).

Reply:
> reg your last paragraph, system updates from my perspective doesn't make sense to say are cold or warm, since we can't charge anyone for that?

yes I mean whateever system updates access warms for the txs. With this logic only the last entry in the ring buffer would be warmed and others cold so not the rest 8191 hashes

Reply:
we decided to not warm anything.